src/trans-word-level/sequence.h

```diff
@@ -19,12 +19,27 @@ class sequence_matcht
 {
 public:
   sequence_matcht(mp_integer __end_time, exprt __condition)
-    : end_time(std::move(__end_time)), condition(std::move(__condition))
+    : _is_empty_match(false),
+      end_time(std::move(__end_time)),
+      condition(std::move(__condition))
   {
   }
 
+  bool empty_match() const
+  {
+    return _is_empty_match;
+  }
+
+  bool _is_empty_match;
```

Comment:
This should be marked `protected` or `private`. (The same is true of the fields below, but those would need to get accessors.)